phpstan.neon

```diff
@@ -1,5 +1,8 @@
+includes:
+    - dev-tools/vendor/phpstan/phpstan-phpunit/extension.neon
```

Comment:
This extension helps PHPStan understand how PHPUnit works. Specifically it helps it know that `static::markTestSkipped()` ends the test execution.